java/pgv-java-grpc/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>pgv-java</artifactId>
         <groupId>io.envoyproxy.protoc-gen-validate</groupId>
-        <version>0.6.8-SNAPSHOT</version>
+        <version>0.6.9-rc.0-SNAPSHOT</version>
```

Comment:
Seems like a pain to manually do. What if we overwrite this via the python script since it's just a quick file io in CI?

Reply:
absolutely, i want to do this as part of a semver step I'll work on next.